cookies.py

```diff
@@ -0,0 +1,45 @@
+import pickle
+import pprint
+import time
+
+from selenium import webdriver
+
+## Using selenium to bypass JSTOR Login 
+
+
+def save_cookies(driver, location):
+
+    pickle.dump(driver.get_cookies(), open(location, "wb"))
+
+
+def load_cookies(driver, location, url=None):
+
+    cookies = pickle.load(open(location, "rb"))
+    driver.delete_all_cookies()
+    # have to be on a page before you can add any cookies, any page - does not matter which
+    driver.get("https://vula.uct.ac.za" if url is None else url)
+    for cookie in cookies:
+        if isinstance(cookie.get('expiry'), float):#Checks if the instance expiry a float 
+            cookie['expiry'] = int(cookie['expiry'])# it converts expiry cookie to a int 
+        driver.add_cookie(cookie)
+
+
+def delete_cookies(driver, domains=None):
+
+    if domains is not None:
+        cookies = driver.get_cookies()
+        original_len = len(cookies)
+        for cookie in cookies:
+            if str(cookie["domain"]) in domains:
+                cookies.remove(cookie)
+        if len(cookies) < original_len:  # if cookies changed, we will update them
+            # deleting everything and adding the modified cookie object
+            driver.delete_all_cookies()
+            for cookie in cookies:
+                driver.add_cookie(cookie)
+    else:
+        driver.delete_all_cookies()
+
+
+# Path where you want to save/load cookies to/from 
```

Comment:
should this path not be independent, right now it's hard coded to your directory?

Reply:
The solution only works on my computer for now. Next sprint I'm looking into generalising it.